Hogajama/hogajama-rs/src/main/java/com/gepardec/hogarama/rest/ActorApiImpl.java

```diff
@@ -4,12 +4,12 @@
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.SecurityContext;
 
-import com.gepardec.hogarama.domain.watering.ActorService;
+import com.gepardec.hogarama.domain.watering.ActorControlService;
 
 public class ActorApiImpl implements ActorApi {
```

Comment:
The same class name and interface name as below, but different meaning. We should clean this up sometimes.